modules/recast/navigation_mesh_editor_plugin.cpp

```diff
@@ -103,24 +103,24 @@ void NavigationMeshEditor::_bind_methods() {
 NavigationMeshEditor::NavigationMeshEditor() {
 
 	bake_hbox = memnew(HBoxContainer);
+
 	button_bake = memnew(ToolButton);
-	button_bake->set_text(TTR("Bake!"));
+	bake_hbox->add_child(button_bake);
 	button_bake->set_toggle_mode(true);
-	button_reset = memnew(Button);
-	button_bake->set_tooltip(TTR("Bake the navigation mesh.") + "\n");
+	button_bake->set_text(TTR("Bake Navigation Mesh"));
 
-	bake_info = memnew(Label);
-	bake_hbox->add_child(button_bake);
+	button_reset = memnew(ToolButton);
 	bake_hbox->add_child(button_reset);
+	button_reset->set_tooltip(TTR("Clear the navigation mesh."));
+	button_bake->connect("pressed", this, "_bake_pressed");
```

Comment:
Since you're reordering statements, I'd put this one close to the `button_bake` ones above too.

Reply:
Missed that one, thanks for noticing.